### ISSUE TITLE
Horizon Client 5.x and shutdown procedure changing

### ISSUE THREAD

**Issue by mcsn747** [Thinstation/thinstation#452]
Hello!
I repeat my question from #441.

I have an issue on Horizon Client 5.0 and succeeding versions.

The shutdown procedure has changed. In version 4.x, after exiting the session, the user sees "Click yes to reconnect" window. In version 5.x, after exiting the session, the user returns to the pool selection window. To complete the work, the user must click File - Quit - Click yes to reconnect - No. If he doesn't, another person can use his saved credentials for login.

I read release notes for VMware Horizon client for Linux and found no related changes.

I have a Ubuntu machine with VMware Horizon client. All client versions - 4.x and 5.x - after exiting the session returns user to the pool selection window. Hence, the described behavior was specific to Thinstation.

How can I return old behavior from version 4.x?

**Comment by Thinstation** [comment]
I have not been able to look at this. I don't have a test environment for the VMWare View Client. Nothing changed in the ThinStation, the behavior that your describing has something to do with the client. Perhaps they broke that behavioral component out from another more complex state request. Have you looked at the cli arguments?

**Comment by mcsn747** [comment]
I have an error during VMware client install
```
Extracting VMware Installer...done.
Component Name Component Long Name Component Version
================================== ============================================================ ====================
vmware-installer VMware Installer 2.1.0.2040127
vmware-horizon-mmr VMware Horizon Multimedia Redirection (MMR) 5.1.0.13956721
vmware-horizon-smartcard VMware Horizon Smart Card 5.1.0.13956721
vmware-horizon-rtav VMware Horizon Real-Time Audio-Video 5.1.0.13956721
vmware-horizon-integrated-printing VMware Horizon Integrated Printing 5.1.0.13956721
vmware-horizon-pcoip VMware Horizon PCoIP 5.1.0.13956721
vmware-horizon-usb VMware Horizon USB Redirection 5.1.0.13956721
vmware-horizon-serialportclient VMware Horizon SerialPort Redirection 5.1.0.13956721
An unknown error occurred during installation. Please check the
installation log at /var/log/vmware-installer for more information.
vmview: Setup Complete
```
/var/log/vmware-installer is missing. What should I change in the install script to save the installation log?

**Comment by Thinstation** [comment]
I would set a break after the chroot, before the cleanup. You should be able to inspect any logs.

**Comment by mcsn747** [comment]
I'm don't cleanup chroot after build.

**Comment by Thinstation** [comment]
You asked what to change in the install script. It makes another chroot and does it’s own clean up. Go set and exit before the cleanup(in the script), it will break the build, but let you look at the logs.

**Comment by mcsn747** [comment]
Ok, Now I'm understand.
Now I have installer log, it has some errors. But I don't see how it relates to my issue.
[vmware-installer.zip](https://github.com/Thinstation/thinstation/files/3600239/vmware-installer.zip)

**Comment by rohrbachger** [comment]
I can confirm this issue, on the 4 Version we did not had this in Thinstation.
Will look at this also, I remember there were some changes for the 5er Versions in Thinstation,
I remember problems with USB.

Here seems to be a solution for an older client, 
https://communities.vmware.com/thread/534673

command line option –hideClientAfterLaunchSession True/False 
or
to add the line view.once = "TRUE" to /etc/vmware/view-default-config. 

I have installed the 5.2 client now, with view.once = "TRUE" in
/root/.vmware-preferences it has worked. as a first test.

Probably we need to define a new war in the 50view conf file 
and add also the changes in the script building the vmware-preferences file.
Maybe this file is not the right place, I noticed it is rewritten after a session.

This is a challenge for me, newer did...
